.travis-command-ea-builds.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euo pipefail
+set +u
```

Comment:
I don't know too much about bash, but would you want to keep the `set -e` part?

Reply:
Fixed.